### ISSUE TITLE
How to get the missing rows to show?

### ISSUE THREAD

**Issue by songstella** [kumkee/solarterms#3]
![Screenshot 2023-01-27 110253](https://user-images.githubusercontent.com/104510788/214977551-474a9d76-1508-47cb-aeef-b36adf26c07f.jpg)

**Comment by kumkee** [comment]
I can't reproduce this, but try [the latest update](https://github.com/kumkee/solarterms/commit/a10566fed5d509424bb815d71cdd364782698937) I just made.
Please do let me know if it works.

**Comment by songstella** [comment]
Thanks! Unfortunately I didn't have much luck ... and sadly I have little to no experience with code so I'm not too good at helping myself. Attached is the cmd prompt log in case it's useful at all. Cheers 
[issue3.txt](https://github.com/kumkee/solarterms/files/10515589/issue3.txt)

**Comment by kumkee** [comment]
> 

Have you updated the code?
If so, `test.py` should have the following line
```
pd.set_option('display.max_rows', 30)
```

**Comment by songstella** [comment]
Yes, double-checked and it's the up-to-date code. Hmm, maybe it's a Windows thing? I also noticed that the term names are adjusted for the southern hemisphere in your screenshot but in mine it shows Minor Cold for January etc. - I guess probably a separate issue but I mention it just in case it might also be a Windows thing?)
![image](https://user-images.githubusercontent.com/104510788/215387716-0c8fc8a2-547f-47e8-88a1-77e8bf2620f5.png)
